torchdynamo/logging.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import os
+
+# logging level for dynamo generated graphs/bytecode/guards
+INFO_CODE = 15
```

Comment:
Just call it verbose? Or something? This feels like it overloads the word info needlessly.